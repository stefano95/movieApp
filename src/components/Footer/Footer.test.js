import React from "react";
import renderer from "react-test-renderer";
import Footer from "./Footer";

describe("Footer render", () => {
  it("renders", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
