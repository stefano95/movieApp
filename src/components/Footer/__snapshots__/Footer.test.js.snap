// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render renders 1`] = `
<div
  style={
    Object {
      "bottom": 0,
      "position": "sticky",
    }
  }
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam condimentum volutpat arcu quis molestie. In non tellus egestas, ultricies magna ut, dictum nisi. Nam cursus dolor sem, in convallis diam commodo eu. Mauris at rutrum arcu, pellentesque sollicitudin augue. Quisque placerat libero id mi feugiat, ut faucibus tortor tempor. Nullam eu neque vitae ante fermentum dictum sed at nisl. Etiam vulputate convallis vehicula. Mauris semper tempor diam eget malesuada. Aenean a consequat purus. Aenean nec odio mi. Fusce a tincidunt quam. Nunc rhoncus ex at ex bibendum dictum. In ullamcorper, enim vel efficitur faucibus, nibh risus sagittis tellus, vitae ultrices lorem dui ut velit. Nam lobortis interdum erat. Aenean a consequat purus. Aenean nec odio mi. Fusce a tincidunt quam. Nunc rhoncus ex at ex bibendum dictum. In ullamcorper, enim vel efficitur faucibus, nibh risus sagittis tellus, vitae ultrices lorem dui ut velit. Nam lobortis interdum erat.
</div>
`;
